### ISSUE TITLE
cfast limit 100 rooms

### ISSUE THREAD

**Issue by AleksanderKamienski** [aamks/aamks#82]
Cfast is limited to 100 rooms. There is a problem with running cfast simulation when the building is bigger and has more rooms. You can separate rooms with an object of type obst but it's not a perfect solution.

**Comment by kowalskiw** [comment]
I am not sure if we can solve this problem. It is just CFAST limitation and we should respect it in our models as well. I don't see any reasonable workarounds to that problem either at the moment.

Compartmentalizing with obstacles will not affect fire environment at all - thus it don't seem to be a comprehensive solution.

Maybe limiting the number of compartments modeled in CFAST can be the way? Using importance sampling principles we can narrow down the geometrical model only to areas which smoke is supposed to reach. We can exclude closed, empty or distant (how much?) rooms from the model at all, because those areas will not affect the result at all. However, this is a tough puzzle to do it right.

_Notice that this and some other limitations of the fire model one can find in CFAST User Guide in Appendix A: https://pages.nist.gov/cfast/manuals.html_

**Comment by kowalskiw** [comment]
According to our last meeting: number of rooms can be changed in our compilation of CFAST (see https://github.com/firemodels/cfast/issues/654) if needed.

**Comment by AleksanderKamienski** [comment]
/usr/local/aamks/fire/cfast_775-1000-i and /usr/local/aamks/fire/cfast_775-1000 do not work for models with more than 100 compartments

**Comment by kowalskiw** [comment]
Indeed, I uploaded wrong versions as `fire.cfast_775-1000` and `fire.cfast_775-1000-i`.
Not diving to deep into CFAST code I've compiled under GNU and Intel up to 750 compartments. However, Intel seems to compile also larger numbers smoothly.

To keep both versions consistent I pull-requested with 750 versions: https://github.com/aamks/aamks/pull/242

**Comment by kowalskiw** [comment]
Resolved since https://github.com/aamks/aamks/commit/25d1483599b2934c1b01989a6821ae43141a96ea
